src/backend/InvenTree/common/serializers.py

```diff
@@ -608,6 +616,194 @@ class Meta:
     result = serializers.CharField()
 
 
+class InvenTreeImageSerializer(
+    InvenTree.serializers.RemoteImageMixin, InvenTreeModelSerializer
+):
+    """Serializer for InvenTreeImage."""
+
+    image = InvenTreeImageSerializerField(required=False)
+    thumbnail = serializers.CharField(source='get_thumbnail_url', read_only=True)
+
+    # we accept the model name here
+    content_type = serializers.ChoiceField(
```

Comment:
Have a look at the `ContentTypeField` class which was introduced in https://github.com/inventree/InvenTree/pull/10699 to address this in a generic manner.

```python
    model_type = ContentTypeField(
        mixin_class=InvenTreeParameterMixin,
        choices=common.validators.parameter_model_options,
        label=_('Model Type'),
        default='',
        allow_null=False,
    )
```

This allows you to specify which class mixin it should be restricted to, and provides robust serialization / de-serialization. It will also dynamically generate the correct list of options.